tests/framework/pkg/schemas/cluster.x-k8s.io.machines/v1beta1/schema.go

```diff
@@ -2,8 +2,7 @@ package schema
 
 import (
 	"github.com/rancher/norman/types"
-	provisioningSchema "github.com/rancher/rancher/tests/framework/pkg/schemas/provisioning.cattle.io/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	m "github.com/rancher/norman/types/mapper"
```

Comment:
This import alias isn't very descriptive but I can't think of a better name `normanTypesMapper` ? 
Im not sure what's the best option to follow conventions (if any) here or just leave it as `m`

Reply:
Yeah I can definitely do that. I'll do `normanMapper`

Reply:
Hey Izaac I can changed this, but this is how the rest of the rancher/rancher repo has the alias for the mapper import, in the other schemas. I think because of the precedent of the rest of the repo, I think it's best to just leave it as `m`

Reply:
I didn't know that and is a good point let's leave it as `m` for consistency